data/json/items/armor/bandolier.json

```diff
@@ -149,6 +213,28 @@
     "use_action": { "type": "bandolier", "capacity": 4, "ammo": [ "40mm" ], "draw_cost": 20 },
     "flags": [ "WATER_FRIENDLY", "BELTED", "OVERSIZE" ]
   },
+  {
+    "id": "stone_pouch",
+    "type": "ARMOR",
+    "name": "stone pouch",
+    "name_plural": "stone pouches",
+    "description": "A medium sized pouch for storing rocks with straps for attaching it to your belt or other webbing.",
+    "weight": "90 g",
+    "volume": "250 ml",
+    "price": 5900,
+    "price_postapoc": 9900,
+    "rigid": false,
+    "material": [ "cotton", "plastic" ],
```

Comment:
Shouldn't contain "plastic" material either, if plastic isn't actually used for crafting.